src/test/java/org/jenkinsci/plugins/durabletask/PowershellScriptTest.java

```diff
@@ -124,7 +124,7 @@
     }
     
     @Test public void implicitError() throws Exception {
-        Controller c = new PowershellScript("MyBogus-Cmdlet").launch(new EnvVars(), ws, launcher, listener);
+        Controller c = new PowershellScript("$ErrorActionPreference = 'Stop'; MyBogus-Cmdlet").launch(new EnvVars(), ws, launcher, listener);
```

Comment:
does this mean that people have to change all their scripts?

Reply:
No. No changes should be required here. The issue is that regardless of the value of $ErrorActionPreference or the ErrorAction parameter provided by all cmdlets a job would fail even if the error was non fatal.  Fatal errors should be controlled via ErrorAction, and this is the standard practice in PowerShell.  So, this is definitely a bug.